RFS/src/main/java/com/rfs/common/Uid.java

```diff
@@ -10,9 +10,9 @@
  * See: https://github.com/elastic/elasticsearch/blob/6.8/server/src/main/java/org/elasticsearch/index/mapper/Uid.java#L32
  */
 public class Uid {
-    private static final int UTF8 = 0xff;
-    private static final int NUMERIC = 0xfe;
-    private static final int BASE64_ESCAPE = 0xfd;
+    public static final int UTF8 = 0xff;
+    public static final int NUMERIC = 0xfe;
+    public static final int BASE64_ESCAPE = 0xfd;
```

Comment:
Could you add another word or two to make these a bit more self-documenting?  I have some guesses what they're for, but I'm probably wrong.

Reply:
I honestly don't understand the \*why\* of these variables myself; per the comment above, this file is straight from the Elastic codebase.